package.json

```diff
@@ -1,6 +1,9 @@
 {
   "private": true,
   "devDependencies": {
-    "ws": "~0.4.28"
+    "ws": "0.4.32"
```

Comment:
Should we leave the 'ws' version alone and upgrade that as a separate change?

Reply:
That's ok, dropped this.